charts/argo-cd/README.md

```diff
@@ -252,7 +256,6 @@ NAME: my-release
 | configs.knownHostsAnnotations | object | `{}` | Known Hosts configmap annotations |
 | configs.repositories | object | `{}` | Repositories list to be used by applications |
 | configs.repositoriesAnnotations | object | `{}` | Annotations to be added to `configs.repositories` Secret |
-| configs.repositoryCredentials | object | `{}` | *DEPRECATED:* Instead, use `configs.credentialTemplates` and/or `configs.repositories` |
```

Comment:
I think these features are deprecated in argo-cd itself but still are supported.
But I am fine with removing them.

Reply:
These are still supported but users were advised to migrate 6 months ago. Even upstream documentation no longer have samples for these.

Reply:
Same as @mkilchhofer I'm happy to remove this, but if it's still a supported, perhaps we could keep it a bit longer for compatibility? Happy to hear other thought on this one.

Reply:
If upstream documentation no longer have samples and advised migration 6 months ago, I think enough time has passed, correct?  I'm good with removal.